fspiop-api/documents/Tracing v1.0.md

```diff
@@ -0,0 +1,895 @@
+
+> traceparent: 00-91e502e28cd723686e9940bd3f378f85-**`06a441483c4f238e`**-01<br/>
+> tracestate: **`mobilemoney`**=**`06a441483c4f238e`**,moja=379cc3faa566cf90,banknrone=0c3e9f44f921ca06<br/>
+
+<br/>
+The request sample including trace headers:
+
+```http
+PUT /quotes/7c23e80c-d078-4077-8263-2c047876fcf6 HTTP/1.1
+Content-Type: application/vnd.interoperability.quotes+json;version=1.0
+Content-Length: 1802
+Date: Tue, 15 Nov 2017 10:13:41 GMT
+FSPIOP-Source: MobileMoney
+FSPIOP-Destination: BankNrOne
+traceparent: 00-91e502e28cd723686e9940bd3f378f85-06a441483c4f238e-01
+tracestate: mobilemoney=06a441483c4f238e,moja=379cc3faa566cf90,banknrone=0c3e9f44f921ca06
+{
+    "transferAmount": {
+        "amount": "99",
+        "currency": "USD"
+    },
+    "payeeReceiveAmount": {
+        "amount": "100",
+        "currency": "USD"
+    },
+    "expiration": "2017-11-15T14:17:09.663+01:00",
+    "ilpPacket": "AQAAAAAAACasIWcuc2UubW9iaWxlbW9uZXkubXNpc2RuLjEyMzQ1Njc4OY-
+IEIXsNCiAgICAidHJhbnNhY3Rpb25JZCI6ICI4NWZlY-
+WMyZi0zOWIyLTQ5MWItODE3ZS00YTAzMjAzZDRmMTQiLA0KICAgICJxdW90ZUlkIjogIjdjMjNlOD-
+BjLWQwNzgtNDA3Ny04MjYzLTJjMDQ3ODc2ZmNmNiIsDQogICAgInBheWVlIjogew0KICAgICAgI-
+CAicGFydHlJZEluZm8iOiB7DQogICAgICAgICAgICAicGFydHlJZFR5cGUiOiAiTVNJU0ROIiwNCiAgI-
+CAgICAgICAgICJwYXJ0eUlkZW50aWZpZXIiOiAiMTIzNDU2Nzg5IiwNCiAgICAgICAgI-
+CAgICJmc3BJZCI6ICJNb2JpbGVNb25leSINCiAgICAgICAgfSwNCiAgICAgI-
+CAgInBlcnNvbmFsSW5mbyI6IHsNCiAgICAgICAgICAgICJjb21wbGV4TmFtZSI6IHsNCiAgICAgICAgI-
+CAgICAgICAiZmlyc3ROYW1lIjogIkhlbnJpayIsDQogICAgICAgICAgICAgICAgImxhc3ROYW1lIjogIk-
+thcmxzc29uIg0KICAgICAgICAgICAgfQ0KICAgICAgICB9DQogICAgfSwNCiAgICAicGF5ZXIi-
+OiB7DQogICAgICAgICJwZXJzb25hbEluZm8iOiB7DQogICAgICAgICAgICAiY29tcGxleE5hbWUi-
+OiB7DQogICAgICAgICAgICAgICAgImZpcnN0TmFtZSI6ICJNYXRzIiwNCiAgICAgICAgICAgICAgI-
+CAibGFzdE5hbWUiOiAiSGFnbWFuIg0KICAgICAgICAgICAgfQ0KICAgICAgICB9LA0KICAgICAgI-
+CAicGFydHlJZEluZm8iOiB7DQogICAgICAgICAgICAicGFydHlJZFR5cGUiOiAiSUJBTiIsDQogICAgI-
+CAgICAgICAicGFydHlJZGVudGlmaWVyI-
+jogIlNFNDU1MDAwMDAwMDA1ODM5ODI1NzQ2NiIsDQogICAgICAgICAgICAiZnNwSWQiOiAiQmFua05yT25
+lIg0KICAgICAgICB9DQogICAgfSwNCiAgICAiYW1vdW50Ijogew0KICAgICAgICAiYW1vdW50IjogIjEw-
+MCIsDQogICAgICAgICJjdXJyZW5jeSI6ICJVU0QiDQogICAgfSwNCiAgICAidHJhbnNhY3Rpb25UeXBlI-
+jogew0KICAgICAgICAic2NlbmFyaW8iOiAiVFJBTlNGRVIiLA0KICAgICAgICAiaW5pdGlhdG9yI-
+jogIlBBWUVSIiwNCiAgICAgICAgImluaXRpYXRvclR5cGUiOiAiQ09OU1VNRVIiDQogICAgfSwNCiAgI-
+CAibm90ZSI6ICJGcm9tIE1hdHMiDQp9DQo\u003d\u003d",
+    "condition": "fH9pAYDQbmoZLPbvv3CSW2RfjU4jvM4ApG_fqGnR7Xs"
+}
+```
+
+4.1.2.4. Quote response callback from Switch to BankNrOne
+
+Switch receives the request, and mutates the trace information before forwarding the **PUT /quotes** callback to BankNrOne.
+
+**Note:** See previous section [4.2.1. Request Party Information](#421-request-party-information) for more detail.
```

Comment:
The link seems to be incorrect, I would guess that it should be to 4.1.1.2. Request Party Information from Switch to MobileMoney?

Reply:
Fixed. It's meant to refer to "4.1.1. Request Party Information" in general.